playbooks/igtf.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-- hosts: all
+- hosts: packstack
   become: yes
   become_method: sudo
   tasks:
```

Comment:
In a later version, I would suggest re-using a role for this - AAROC.certificates or something from egi-qc

Reply:
Yes we need to have a dedicated role for this, for sure.

Reply:
Ok, done in #8